Nos/Models/Profile.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Profile.swift
+//  Nos
+//
+//  Created by Martin Dutra on 27/4/23.
+//
+
+import Foundation
+import secp256k1
+
+/// A struct that abstracts the information included ina nprofile string.
+///
+/// See https://github.com/nostr-protocol/nips/blob/master/19.md
+/// In the future we can parse and include the relays in this struct, but now we just need the public key.
+struct Profile {
+
+    var publicKey: PublicKey
+
+    var hex: String {
```

Comment:
Can we call this `publicKeyHex` because `hex` makes it sound like it could be the nprofile hex or something.

Reply:
👌